hw3/hw3_task01.py

```diff
@@ -0,0 +1,60 @@
+"""
+In previous homework task 4, you wrote a cache function that remembers other function output value.
+
+Modify it to be a parametrized decorator, so that the following code:
+
+@cache(times=3)
+def some_function():
+    pass
+
+Would give out cached value up to times number only. Example:
+
+@cache(times=2)
+def f():
+    return input('? ')   # careful with input() in python2, use raw_input() instead
+
+>> f()
+? 1
+'1'
+>> f()     # will remember previous value
+'1'
+>> f()     # but use it up to two times only
+'1'
+>> f()
+? 2
+'2'
+
+
+"""
+import pickle  # noqa: S403
+from typing import Any, Callable
+from unittest.mock import MagicMock
+
+
+class TestPlug:
+    """Test plug for class object."""
+
+    pass
+
+
+mocker = MagicMock()
```

Comment:
Why you need Mock in this module?  Will tests fail if you remove it?

Reply:
I'm using Mock for call_count.

Reply:
This is not a question about why you using Mock, it is a question of why are you using it here, not in tests?